src/emulate.c

```diff
@@ -40,14 +40,25 @@ extern struct target_ops gdbstub_ops;
     _(breakpoint)       /* Breakpoint */                     \
     _(load_misaligned)  /* Load address misaligned */        \
     _(load_fault)       /* Load access fault */              \
-    _(store_misaligned) /* Store/AMO address misaligned */
+    _(store_misaligned) /* Store/AMO address misaligned */   \
+    _(store_fault)      /* Store/AMO access fault */         \
+    _(ecall_U)          /* Environment call from U-mode */   \
+    _(ecall_S)          /* Environment call from S-mode */   \
+    _(reserved)         /* Reserved */                       \
```

Comment:
Shall we care about "Reserved" item?

Reply:
I think the "reserved" item should be removed. Maybe I need to
rewrite the exception list by assigning exception code and remove
unuseful items.

Reply:
In addition, check ecall_{U,S} as well since only `ecall_M` makes sense now.